sdk/eventhubs/azure-messaging-eventhubs/test/ut/processor_test.cpp

```diff
@@ -121,7 +121,7 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace Test {
       producerOptions.Name = "Producer for LoadBalancerTest";
       ProducerClient producerClient{connectionString, eventHubName, producerOptions};
 
-#if PROCESSOR_ON_TEST_THREAD
+#if TRUE
```

Comment:
`TRUE` is a macro that is defined in `windows.h`, correct?
```suggestion
#if 1
```

Reply:
The code comp;iles without windows.h, so it's defined somewhere.

But it's irrelevant. That was a test #ifdef and should ahve been removed.